docs/source/components.rst

```diff
@@ -240,6 +261,12 @@ Cross field validation is made possible by overriding ``Form.clean()``:
             return data
 
 
+If you use a form's cleaned data to create a new model then model validation
+will also take place. It can be customized by passing a list of validators to a
```

Comment:
Yes, you're right. For some reason I assumed that `Model.save()` validates the model, but it doesn't.